src/main.ts

```diff
@@ -71,13 +71,18 @@ export async function main({
     outputLogger,
   });
 
-  await lintProjects({
+  const isAllPassed = await lintProjects({
```

Comment:
Nit: What do you think about calling this `isAllSuccessful` instead of `isAllPassed`, since `isAllPassed` in `src/report-project-lint-result.ts` strictly means that there were no failing rules, but in this case it means not only that, but also no errors were thrown?